docs/getting-started/installation.md

```diff
@@ -0,0 +1,21 @@
+# Installing Tools
+
+Regardless of your language, you need to make sure you install the [Game Tools](https://docs.wpilib.org/en/stable/docs/zero-to-robot/step-2/frc-game-tools.html) (Windows only) and [WPILib](https://docs.wpilib.org/en/stable/docs/zero-to-robot/step-2/wpilib-setup.html), selecting "Everything" and "Skip and don't use VS Code", VS Code is not the best tool for developing with Kotlin, that title lies firmly with IntelliJ, which we will discuss next.
+
+## IntelliJ
+
+The tool of choice for using Kotlin in FRC is IntelliJ. You'll want to start by installing IntelliJ Community (it's free, so don't worry).
+
+Once you install IntelliJ you'll want to open `File > Settings > Plugins` and then click "Marketplace". Once you've opened the plugin marketplace, search for the "FRC" plugin by Javaru, then click install.
```

Comment:
I don't see 'Javaru' when searching for the plugin.

<img width="1103" alt="image" src="https://github.com/user-attachments/assets/de4780aa-a6f6-4d28-a454-85910eab11a5">

Reply:
I'll fix that, it seems to show up different for different people? because in the past I've seen Mark Vedder too, I'll probably just make it day like "By Javaru or Mark Vedder"